etc/gensbom/gensbom.sh

```diff
@@ -71,6 +71,13 @@ function usage() {
 	    A file with the valid container registry credentials following the
 	    Docker format. This file must be in the current working directory
 
+	  ${_VIOLET}ca-bundle.crt${_RESET}
+	    SSL/TLS certificates bundle. If the TPA instance uses SSL/TLS, this
```

Comment:
This is only required if trustify uses a certificate signed by a custom trust anchor. Not in general when using TLS.